keyboards/lets_split/keymaps/piemod/keymap.c

```diff
@@ -109,7 +109,7 @@ int CAPSLOCKED = 0;
     [3] =  ACTION_LAYER_TAP_KEY(3, KC_E),     // FN3 = Momentary numbEr layer on E.
     [4] =  ACTION_LAYER_TAP_KEY(4, KC_U),     // FN4 = Momentary fUnction layer on U.
     [5] =  ACTION_LAYER_TAP_KEY(5, KC_Q),     // FN5 = Momentary emaQs layer on Q.
-    [6] =  ACTION_LAYER_TAP_KEY(6, KC_LCTRL), // FN6 = Momentary MACROS on ??? key.
+    [6] =  ACTION_LAYER_TAP_KEY(6, KC_LCTRL), // FN6 = Momentary COMBOS on ??? key.
```

Comment:
If you want, you could use `LT(x, kc)` in the keymap instead.

This accomplishes the same thing, but may look a bit cleaner.